packages/mdx/src/mini-editor/index.scss

```diff
@@ -102,4 +104,17 @@
   min-width: 1.5em;
   min-height: 1.5em;
   margin-right: 0.8em;
+
+  &[type="button"] {
+    background-color: transparent;
+    border: 1px solid transparent;
+    color: inherit;
+  }
+
+  svg {
+    width: 1.5em;
+    height: 1.5em;
+    min-width: 1.5em;
+    min-height: 1.5em;
+  }
```

Comment:
I can see that the copy button is also used with other classes like `.ch-code-button`.
While you can copy these reset styles into the appropriate scss file, another option would be to add the styles directly yo the button on the `copy-buton.tsx` file. Something like this:

```
<button
  style={{
    background: "none",
    border: 0,
    color: "inherit",
    fontSize: "inherit",
    padding: 0,
    ...style,
  }}
  ...
```